src/main/java/mate.academy/lib/Injector.java

```diff
@@ -1,13 +1,58 @@
 package mate.academy.lib;
 
+import java.lang.reflect.Field;
+import mate.academy.storage.ImplementationStorageImpl;
+import mate.academy.storage.InstanceStorageImpl;
+import mate.academy.storage.Storage;
+
 public class Injector {
     private static final Injector injector = new Injector();
+    private static Storage<Class<?>, Class<?>> implementations
+            = new ImplementationStorageImpl();
+    private static Storage<Class<?>, Object> instances
+            = new InstanceStorageImpl();
```

Comment:
for future: there is no actual need to wrap java collections into some oop form
we can use them in the original form (map, list, so on)